### DISCUSSION TITLE
Would it be possible to list known users of the library?

### DISCUSSION THREAD

**Discussion by Kimi-Arthur** [CoderLine/alphaTab#687] [Q&A]
I've seen usage of this lib in https://www.gtpso.com. But the website is not as good as the lib itself.

So I'm wondering whether there are more sites using this, but could not find out.

**Comment by Danielku15** [comment]
I once started for me personally a list where the I know the library is used, but it's not always easy to find out. There would be https://github.com/CoderLine/alphaTab/discussions/categories/show-and-tell where I hoped people announce that they use it. I am always grateful for any insights on where alphaTab is used, the more it is used, the more motivated I might be to continue making nice features. 

Unfortunately often alphaTab is also used behind a pay-wall as it is used on sites where you need subscriptions. I would love to add a section on the website showing how alphaTab is used in the real world.

**Reply by Danielku15** [reply]
Here some of the products and sites I know from partnerships and reported issues: 
- https://gprotab.net/ 
- https://www.infiniteguitar.com/
- https://fretello.com/
- https://mlhoutel.github.io/Tablatures/#/
- https://www.emotive.productions/
- https://broken-guitar.ir/

**Reply by Kimi-Arthur** [reply]
That's a great list to start with. Thanks a lot!

However, if what I want is a local editor to write AlphaTex and generate pdf or interactive page, is there a good solution that you know of?

**Reply by Danielku15** [reply]
I wouldn't know that somebody built a software like that using alphaTab. As you might know: alphaTab is primarily an SDK/Library for others to build music notation software. It's not an own Music Notation Software Suite so sombody would have to build this.
